### DISCUSSION TITLE
Only one Pump Fan appearing

### DISCUSSION THREAD

**Discussion by XenosVita** [Rem0o/FanControl.Releases#2063] [Q&A]
Has anyone encountered this or similar wherein a mobo has more than one Pump Fan pwm input (in this case just the 2 total) and Fan Control is able to propagate Pump Fan 1 and not Pump Fan 2? It can read all the other Sys_Fan inputs (even empty ones).

Yes, all inputs are set to PWM in BIOS and yes both are powered and functioning; MSI Center's hardware monitor confirms both.

**Comment by Toofast520** [comment]
I'm also having this issue and wondered if you resolved it? I'm running an MSI Godlike board with two pump headers but only one shows in fan control, but both show in MSI Center
